pkg/launcher/launcher.go

```diff
@@ -136,10 +136,12 @@ func Launch(params Parameters) {
 	}
 
 	// Add the Metrics Service
-	if !strings.Contains(namespace, ",") {
-		addMetrics(ctx, cfg, namespace)
-
+	if operatorNs, err := k8sutil.GetOperatorNamespace(); err == nil {
+		addMetrics(ctx, cfg, operatorNs)
+	} else {
+		log.Info("Could not get OperatorNamespace. Not in cluster? Skipping addMetrics")
```

Comment:
Seems to be `serveCRMetrics` do the same

Reply:
What do you me @dmvolod?

Reply:
We can modify code like this:
```
if !strings.Contains(namespace, ",") {
	addMetrics(ctx, cfg, namespace)
} else {
	serveCRMetrics(cfg)
}
```
wdyt?

Reply:
@dmvolod I don't know. We're skipping all addMetrics code after line:
https://github.com/infinispan/infinispan-operator/blob/e055d4d751202485ed3fde908c0cfb6449c43cfe/pkg/launcher/launcher.go#L170

is that ok?

Reply:
Yeah, probably true
Need to be test with different options

Reply:
So is it not possible to have operator metrics when using multi-namespace? Not a blocker obviously, but that doesn't sound ideal.

Reply:
@ryanemerson I've checked with a manual deploy: current implementation is ok, metrics service is deployed in the namespace where the operator is running